package.json

```diff
@@ -36,9 +37,11 @@
     "html-loader": "^0.5.5",
     "html-webpack-plugin": "^3.1.0",
     "style-loader": "^0.20.3",
+    "stylint": "^1.5.9",
+    "stylus": "^0.54.5",
     "url-loader": "^1.0.1",
     "webpack": "^4.2.0",
     "webpack-cli": "^2.0.13",
     "webpack-dev-server": "^3.1.1"
   }
-}
+}
```

Comment:
The linter dont let me leave an empty line at the end of the __css__ and __json__ files. 🤔